### PR TITLE
Lavaland Loot: Cursed heart upgraded to wizard version, New loot item that grants omnitongue

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Why: I used the wrong cursed heart so it actually did nothing, woops!
As for the omnitongue book: the language system is cool

:cl: Kor
add: a new mysterious book has been discovered in lavaland chests!
tweak: weird purple hearts found in lavaland chests have started beating again! what could this mean?
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -343,6 +343,24 @@
...
+
+/obj/item/grant_omnitongue
+	name = "Book of Babel"
+	desc = "An ancient tome written in countless tongues."
+	icon = 'icons/obj/library.dmi'
+	icon_state = "book1"
+	w_class = 2
+
+/obj/item/grant_omnitongue/attack_self(mob/user)
+	to_chat(user, "You flip through the pages of the book, quickly and conviently learning every language in existence. Somewhat less convienently, the aging book crumbles to dust in the process. Whoops.")
```
Comment (kevinz000) [inline code/modules/mining/lavaland/necropolis_chests.dm]: conviently

**Comment by RemieRichards** [comment]
Rude.
Also the only reason no one wants it is because you gave them the version with literally no upside to it! only the wizard one has healing, the default is quite literally a punishment item.

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mining/lavaland/necropolis_chests.dm]: spelt wrong both times.

**Comment by FantasticFwoosh** [comment]
Players have already theorycrafted ways to abuse the cursed heart @RemieRichards  in order to out-tank damage, by rolling a bloodless race using xenobio and going from there. Im not too sad to see it gone.

**Comment by KorPhaeron** [comment]
@RemieRichards 

I wasn't aware there was more than one, my bad. I guess this could replace the cult robes which are worse than your starting armour instead.

**Comment by RemieRichards** [comment]
No, Fwoosh, you don't understand, the one available to non-wizards has NO upsides, it does NOT heal like the wizard version does, there's no advantage at all to it.

**Comment by coiax** [comment]
OMNITONGUE IS NOT GRANT ALL LANGUAGES

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -33,7 +33,7 @@
 		if(9)
 			new /obj/item/organ/brain/alien(src)
 		if(10)
-			new /obj/item/organ/heart/cursed(src)
+			new /obj/item/grant_omnitongue(src)
```
Comment (coiax) [inline code/modules/mining/lavaland/necropolis_chests.dm]: why not just call it /obj/item/book_of_babel

Comment (FantasticFwoosh) [inline code/modules/mining/lavaland/necropolis_chests.dm]: I honestly thought the same thing when i saw the PR, put a picture of the leaning tower of pisa on the front or something

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -343,6 +343,24 @@
 			add_fingerprint(usr)
 
 
+
+
+//Book of Babel
+
+/obj/item/grant_omnitongue
```
Comment (lzimann) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Change the path to book_of_babel or something along these lines.

**Comment by optimumtact** [comment]
Changelog

**Comment by lzimann** [comment]
@korphaeron pls

**Comment by jughu** [comment]
Cursed tongue when

**Comment by ExcessiveUseOfCobblestone** [comment]
Can you add the real cursed heart [or at least a GIMPED version that isn't bad], because I'd take the heart if it wasn't completely useless.

**Comment by RemieRichards** [comment]
@tgstation/commit-access 
Needs a new maintainer to review it now that I've done Dead Kor's work for him.

**Comment by lzimann** [comment]
thanks @RemieRichards

**Pull request merged**